db/migrate/20170914220910_add_has_family_under18_to_adoption_app.rb

```diff
@@ -0,0 +1,5 @@
+class AddHasFamilyUnder18ToAdoptionApp < ActiveRecord::Migration[5.0]
```

Comment:
We've been bad about this, but migrations adding columns should be deployed by themselves so that the model isn't looking for a column that isn't in the DB yet.  That's up to @markottaviani though since he does the deploys.

Reply:
Just curious since I am new to all of this, how would that work? Do we just not do this locally and deploy?

Reply:
@TheDudeWithTheThing is referring to the production deployments.    I believe with the way I have the Capistrano stuff setup the database migrations happen first anyway. However I agree with trying to split out the data model ahead of time as much as we can.